WAKU.md

```diff
@@ -0,0 +1,17 @@
+# Waku spec support
+
+*Last updated February 14, 2020*
+
+status-nim client of Waku is spec compliant with [Waku spec v0.3](https://specs.vac.dev/waku.html) with the exception of:
+- It doesn't support all the MUST packet codes (!)
+- Currently nodes with higher version don't automatically disconnect if versions are different
+
+It doesn't yet implement the following recommended features:
+- It doesn't disconnect a peer if it receives a message before a Status message
```

Comment:
```suggestion
- It doesn't apply a timeout to receive Status message
```

Reply:
This is both about:

>  When a node does not receive the Status message from a peer, before a configurable timeout, it SHOULD disconnect from that peer.

and

> When a node is receiving other Waku messages from a peer before a Status message is received, the node MUST ignore these messages and SHOULD disconnect from that peer

Reply:
The latter is handled. It will disconnect because the current implementation requires to receive Status message first. If it's any other message, a peer will be disconnected.

The first is not implemented that's why I suggested a change.